rest-kraken/src/main/java/com/github/twitch4j/kraken/domain/KrakenCollectionMetadata.java

```diff
@@ -0,0 +1,51 @@
+package com.github.twitch4j.kraken.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Setter;
+
+import java.time.Instant;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class KrakenCollectionMetadata {
+
+    @JsonProperty("_id")
+    private String id;
+
+    @JsonProperty("created_at")
```

Comment:
You can avoid this via @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class) (applies for numerous other fields in this class too)

Reply:
Should I apply this to the classes or will it be fixed in another pr?